storage/mapCacher.go

```diff
@@ -8,7 +8,7 @@ import (
 	logger "github.com/ElrondNetwork/elrond-go-logger"
 )
 
-var log = logger.GetOrCreate("mapCacher")
+var log = logger.GetOrCreate("elrond-sdk-erdgo/storage")
```

Comment:
why did you change mapCacher to storage if the struct name has been kept the same ?

Reply:
this is actually the package: `elrond-sdk-erdgo/storage`. The naming is now consistent in this repo. I've chosen to add the `elrond-sdk-erdgo` in order to better localize the possible problem if we used the sdk in other problem